spec/views/catalog/show.html.erb_spec.rb

```diff
@@ -4,9 +4,13 @@
   let(:document) { SolrDocument.new :id => 'xyz', :format => 'a' }
   let(:title) { 'Long title that should be truncated at 50 characters' }
   let(:blacklight_config) { Blacklight::Configuration.new }
+  let(:query_params) { { controller: 'catalog', action: 'show' } }
+  let(:search_state) { Blacklight::SearchState.new(query_params, blacklight_config) }
   before(:each) do
     assign :document, document
     allow(view).to receive(:blacklight_config).and_return(blacklight_config)
+    allow(view).to receive(:search_state).and_return search_state
+    allow(view).to receive(:application_name).and_return 'Argo'
```

Comment:
This is coming from here: https://github.com/sul-dlss/argo/blob/ac6483915e12ca0cfe4735ca8bab907d962e97c7/config/locales/blacklight.en.yml#L3  I think you should fix that there and not stub out this instance.